CODING_STANDARD.md

```diff
@@ -188,6 +188,10 @@ Formatting is enforced using clang-format. For more information about this, see
 - Avoid `assert`. If the condition is an actual invariant, use INVARIANT,
   PRECONDITION, POSTCONDITION, CHECK_RETURN, UNREACHABLE or DATA_INVARIANT. If
   there are possible reasons why it might fail, throw an exception.
+    - Use "should" style statements for messages in invariants (e.g. "Array
+      should have a non-zero size") to make it clear both the violation and the
```

Comment:
I think it should be "to make clear both the violation ..." (drop "it")

Reply:
Opted for:
> to make both the violation and the expected behavior clear